load-testing/load-test-stress.json

```diff
@@ -0,0 +1,34 @@
+{
+  "objective": "Check if success rate stays above 90%",
+  "details": {
+    "name": "internalstaging.datastage.io",
+    "UWSGI_PROCESSES": 1,
+    "fence_autoscaler": {
+      "min": 20,
+      "max": 25,
+      "targetCpu": 40
+    },
+    "nginx_rate_limit": "6r/s"
+  },
+  "service": "fence",
+  "load_test_scenario": "presigned-url",
+  "presigned_url_guids": ["2d574740-d90f-42af-84e0-e12fdedd17a5"],
+  "indexd_record_url": "",
+  "virtual_users": [
+     { "duration": "30s", "target": 1 },
+     { "duration": "30s", "target": 5 },
+     { "duration": "30s", "target": 10 },
+     { "duration": "60s", "target": 20 },
+     { "duration": "60s", "target": 30 },
+     { "duration": "60s", "target": 40 },
+     { "duration": "60s", "target": 50 },
+     { "duration": "60s", "target": 50 },
+     { "duration": "60s", "target": 50 },
+     { "duration": "60s", "target": 50 },
+     { "duration": "60s", "target": 60 },
+     { "duration": "60s", "target": 70 },
+     { "duration": "60s", "target": 50 },
+     { "duration": "60s", "target": 30 },
+     { "duration": "60s", "target": 0 }
```

Comment:
similarly here. A good scenario to get best out of the service!

Reply:
We can improve the documentation on where we should store sample scenarios.